tests/Aquifer.Common.UnitTests/HtmlUtilitiesTests.cs

```diff
@@ -7,33 +7,25 @@ public sealed class HtmlUtilitiesTests
     public static IEnumerable<object[]> GetHtmlToPlainTextConversionTestData =>
     [
         [
-            (
-                Html: /* lang=html */ """
-                    <body>
-                        <div>
-                            <p>
-                            <!-- This is a comment.-->
-                                Test  &nbsp;   *HTML.
-                            </p>
-                        </div>
-                    </body>
-                    """,
-                PlainText: "Test  \u00a0   *HTML.",
-                WordCount: 2
-            ),
+            (Html: /* lang=html */ """
```

Comment:
These whitespace changes are going to cause conflicts with my refactoring PR: https://github.com/BiblioNexusStudio/aquifer-server/pull/603

I also moved the test class into a `Utilities` folder.